pynance/data/retrieve.py

```diff
@@ -10,14 +10,14 @@
 Wraps `Pandas Remote Data Access 
 <http://pandas.pydata.org/pandas-docs/stable/remote_data.html>`_.
 """
-
+from datetime import datetime
 from ftplib import FTP
 from functools import partial
 import io
 
 import pandas_datareader.data as web
 
-def get(equity, start, end):
+def get(equity, start, end = datetime.today().date().isoformat()):
```

Comment:
For documentation methods see:

- http://stackoverflow.com/questions/12082570/override-function-declaration-in-autodoc-for-sphinx/12087750#12087750
- http://stackoverflow.com/questions/5365684/is-it-possible-to-override-sphinx-autodoc-for-specific-functions